README.md

```diff
@@ -1 +1,77 @@
-# hello-microverse
\ No newline at end of file
+
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# hello-microverse
+
+> This is a site showing hello microverse text as a header.
+
+![screenshot](./images/pic.png)
+
+It has a white background with the header in red color.
+
+## Built With
+
+- html,css
+- Linters,Mark down
+
+
+
+
+## Getting Started
+
+To get a local copy up and running follow these simple example steps.
+
+1)Enter `git clone git@github.com:basnetlaxmi/hello-microverse.git` into terminal
+
+2)Open in browser to view site
+
+
+### Prerequisites
+-code editor
+-git and github
+
+### Setup
+
+### Install
+
+### Usage
+
+### Run tests
+
+### Deployment
+
```

Comment:
Remove "Setup", "Install", "Usage", "Run tests" and "Deployment" sections if you are not using them.

- [ ] Remove empty sections.